src/Microsoft.VisualStudio.Web.CodeGeneration.Templating/Microsoft.VisualStudio.Web.CodeGeneration.Templating.csproj

```diff
@@ -20,7 +20,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Razor" Version="1.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Razor" Version="1.0.2" />
```

Comment:
oops I searched razor and was looking at Mvc.Razor, the dependency injection is 1.0.2 though